File: DailyFlash Solutions/DailyFlash Solutions/74-DailyFlash_Solutions/15_April_Solution_Three/C/Program0.c
#include<stdio.h>

void main(){

	int n = 0;

	printf("Enter Number of Elements : ");
	scanf("%d",&n);

	int arr1[n];
	int arr2[n];
	int arr3[2*n];

	printf("Enter For array 1\n");
	for(int i=0; i<n; i++){

		scanf("%d",&arr1[i]);
	}

	printf("Enter For array 2\n");
	for(int i=0; i<n; i++){

		scanf("%d",&arr2[i]);
	}

	for(int i = 0; i < n; i++){

		arr3[i] = arr1[i];
		arr3[n+i] = arr2[i];
	}

	printf("After Concatenation Third array is\n");
	for(int i=0; i<2*n; i++){

		printf("%d\t",arr3[i]);

	}
	printf("\n");

	for(int i=0; i<2*n; i++){
		for(int j=0; j<2*n; j++){
		
			if(arr3[j] < arr3[i]){

				int temp = arr3[i];
				arr3[i] = arr3[j];
				arr3[j] = temp;
			}
		}
	}

	printf("After Concatenation and Sorting Third array is\n");
	for(int i=0; i<2*n; i++){

		printf("%d\t",arr3[i]);

	}
	printf("\n");

}

File: DailyFlash Solutions/DailyFlash Solutions/74-DailyFlash_Solutions/15_April_Solution_Three/Java/Program0.java
import java.io.*;
import java.util.*;

class ArrayOps{

	public static void main(String ... args)throws IOException{

		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

		System.out.println("Enter No of Elements : ");
		int m = Integer.parseInt(br.readLine());

		int arr1[] = new int[m];
		int arr2[] = new int[m];
		int arr3[] = new int[m*2];


		System.out.println("Enter Elements For Array 1");
		for(int i=0; i<m; i++){

			arr1[i] = Integer.parseInt(br.readLine());
		}

		System.out.println("Enter Elements For Array 2");
		for(int i=0; i<m; i++){

			arr2[i] = Integer.parseInt(br.readLine());
		}

		for(int i = 0; i < m; i++){

			arr3[i] = arr1[i];
			arr3[m+i] = arr2[i];
		}

		System.out.printf("After Concatenation Third array is\n");
		for(int i=0; i<2*m; i++){

			System.out.printf("%d\t",arr3[i]);

		}
		System.out.println();
		
		for(int i=0; i<2*n; i++){
		for(int j=0; j<2*n; j++){
		
			if(arr3[j] < arr3[i]){

				int temp = arr3[i];
				arr3[i] = arr3[j];
				arr3[j] = temp;
			}
		}
	}

System.out.printf("After Concatenation and Sorting Third array is\n");
		for(int i=0; i<2*m; i++){

			System.out.printf("%d\t",arr3[i]);

		}
		System.out.println();

	}
}
